mobile/mobile_46.asm

```diff
@@ -873,7 +873,7 @@ Function118746:
 	dw Function1188c8
 	dw Function11878d
 	dw Function118903
-	dw Function118ad0
+	dw Set_IndexDownloadURL
```

Comment:
OddEggDownloadURL or TamagoDownloadURL

It'd be nice to prefix these functions with some common prefix but right now it's unclear what the different jumptables are for...

Reply:
`OddEggDownloadURL`, please.